<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mrqaxes2</class>
 <widget class="QWidget" name="mrqaxes2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>483</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb_tuning">
     <property name="title">
      <string>Tuning</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_23">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Time Duty</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_22">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Mode</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_24">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Feedback</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QComboBox" name="cmbTuneMode">
        <property name="minimumSize">
         <size>
          <width>76</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>PVT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>无时间LVT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>有时间LVT</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QComboBox" name="cmbTimeWidth">
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>1/4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1/8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1/16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1/32</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QComboBox" name="cmbTuneFb">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Emergency Stop</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mode</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Distance</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmbStopMode">
        <item>
         <property name="text">
          <string>Immediately</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Delay</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cmbStopDist">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cmbStopTime">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Time</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_limit">
     <property name="title">
      <string>Warning</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>Step Loss</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmbWarnLoss">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Default Position</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Unit</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmbInitUnit">
        <item>
         <property name="text">
          <string>Degree</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Angle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IO</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cmbInitIO">
        <item>
         <property name="text">
          <string>TRIG1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TRIG2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TRIG3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TRIG4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TRIG5</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Position</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cmbInitPos">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>IO</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="btnInitZero">
        <property name="text">
         <string>To Default</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
